styles.css

```diff
@@ -515,19 +512,101 @@ form {
 .image-desktop,
 .about-project-desktop,
 .client-name-desktop,
-.name-desktop,
 .li-desktop,
+.project-desktop,
+.desktop-div {
+  display: none;
+}
+
+.popup {
```

Comment:
- [ ] kindly consider giving this pop-up a fixed position so that when you scroll it only scrolls on the page not scrolling even the pop-up to give a good user experience  check the images below for more clarification 
- your design 👇 
<img width="277" alt="Screen Shot 2023-06-22 at 14 00 50" src="https://github.com/agneta1991/Portfolio/assets/60146030/728d002a-eed8-43f7-a152-bde2b66f14b5">